README.md

```diff
@@ -71,10 +63,42 @@ be good to go.
 Building the front-end
 ----------------------
 
-Run `grunt build` to regenerate the front-end logic and css files. For production, 
+Run `grunt build` to regenerate the front-end logic and css files. For production,
 run with `grunt build:prod` to get minified versions, and run `grunt build:prod:true` to enable
 auto building whenever a front-end file changes. See `Gruntfile.js` for details.
 
 **NOTE:** Our front-end is in transition from an older architecture using Requirejs to
 a new one using our build system. Check in our IRC channel (irc.mozilla.org#thimble) for
 the latest state of development.
+
+Bramble code editor
+-------------------
+
+In order to load Thimble, the bramble code editor to be statically served separately.
```

Comment:
Were you going for, *In order to load Thimble the bramble code editor must be served separately*?